packages/app/src/elements/item-icon.ts

```diff
@@ -48,7 +48,10 @@ export class ItemIcon extends LitElement {
             } catch (e) {}
         }
 
-        const favIcon = app.settings.favicons && url ? `https://icons.duckduckgo.com/ip3/${url}.ico` : undefined;
+        const faviconUrl = url?.startsWith("*.") ? url.replace("*.", "") : url;
```

Comment:
This is an unrelated fix, for URLs with the new `*.` format.

Reply:
Can we move this outside of this PR, please? I think it may be a bit longer before we can merge this one.